packages/blockchain-wallet-v4-frontend/src/scenes/Home/PriceIndexSeries/Chart/index.js

```diff
@@ -41,14 +41,28 @@ class Chart extends React.Component {
     const { start, interval } = selectPriceIndexSeriesOptions(coin, timeframe)
```

Comment:
I see ('ls.chart') item in my localStorage, but doesn't look we're using that to load the latest 'settings/config' of the chart.

Reply:
We are using it to load the config of the chart if the selector getPriceIndexSeries returns a Remote.Failure. 
`Failure: (message) => {
        const value = JSON.parse(window.localStorage.getItem('ls.chart-data'))
        return isNil(value)
          ? (<Error>{message}</Error>)
          : (<Success
            coin={coin}
            timeframe={timeframe}
            config={configure(start, interval, currency, value)}
            selectCoin={this.selectCoin}
            selectTimeframe={this.selectTimeframe}
          />)
      },`